README.md

```diff
@@ -1,32 +1,98 @@
-# Switch Statement
+# Switch Statements
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Explain what a switch statement is.
 
-## Introduction
+## What is a Switch Statement?
 
-A 1-2 sentence summary of what will be covered.
+A **switch statement** is a multiway branch that is more efficient than nested
+`if` statements in that it can have multiple execution paths. To help understand
+more about what goes into a switch statement, let us look at the syntax:
 
-## Topic 1
+```java
+switch (expression) {
+        // case statements
+        // values must be the same data type as the expression
+        case value1:
```

Comment:
The indentation looks a little off in GitHub

Reply:
Thanks for the heads up! The linter messed up some formatting when I ran it.